.github/workflows/ci-build.yaml

```
@@ -0,0 +1,60 @@
+name: CI build
+on:
+  pull_request: {}
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: ubuntu:bionic
+    env:
+      repo: connection-limit
+
+    steps:          
+      - name: Setup build environment
```

```suggestion
      - name: Set up build environment
```
nit: "set up" is the verb, "setup" is a noun

---

Done.